mappings/com/mojang/blaze3d/platform/GLX.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS com/mojang/blaze3d/platform/GLX
+	METHOD _getRefreshRate (Lnet/minecraft/class_1041;)I
+		ARG 0 window
+	METHOD _init (IZ)V
+		ARG 0 debugVerbosity
+		ARG 1 debugSync
+	METHOD _renderCrosshair (IZZZ)V
+		ARG 0 size
+		ARG 1 drawX
+		ARG 2 drawY
+		ARG 3 drawZ
+	METHOD _setGlfwErrorCallback (Lorg/lwjgl/glfw/GLFWErrorCallbackI;)V
+		ARG 0 callback
+	METHOD _shouldClose (Lnet/minecraft/class_1041;)Z
+		ARG 0 window
+	METHOD getErrorString (I)Ljava/lang/String;
+		ARG 0 code
+	METHOD make (Ljava/lang/Object;Ljava/util/function/Consumer;)Ljava/lang/Object;
```

Comment:
Hmm, does this sound better than `create` which we have in `Util` (consider changing name there?)